src/Components/Overview/Overview.js

```diff
@@ -0,0 +1,39 @@
+import './Overview.scss';
+
+import { Card, CardBody, CardTitle, Title, TitleSizes } from '@patternfly/react-core/dist/esm/components';
+
+import React from 'react';
+import messages from '../../Messages';
+import routerParams from '@redhat-cloud-services/frontend-components-utilities/files/RouterParams';
+import { useIntl } from 'react-intl';
+
+const OverviewTab = () => {
+    const intl = useIntl();
+
+    const steps = [{
+        title: intl.formatMessage(messages.insightsIsDesigned),
+        body: intl.formatMessage(messages.insightsIsDesignedBody)
+    }, {
+        title: intl.formatMessage(messages.youControlWhat),
+        body: intl.formatMessage(messages.youControlWhatBody)
+    }, {
+        title: intl.formatMessage(messages.dataIsEncrypted),
+        body: intl.formatMessage(messages.dataIsEncryptedBody)
+    }, {
+        title: intl.formatMessage(messages.onlyOneUploaded),
+        body: intl.formatMessage(messages.onlyOneUploadedBody)
+    }];
+    const cardBuilder = card => <Card className='ins-c-card'>
```

Comment:
Maybe throw these in a `Stack` instead of using `900px` to define width?

Reply:
Or maybe just convert from `px` to `%`? 🤔

Reply:
oooo 70% looks good, feels better <3